oci/private/tarball_run.sh.tpl

```diff
@@ -14,4 +13,8 @@ else
     exit 1
 fi
 
+# TODO: some loader implementations don't need a tar input, so this might be wasted time
+readonly IMAGE="$(mktemp -u).tar"
```

Comment:
this will leak, we probably want to remove this file once it's loaded.

Reply:
we could use the `docker load -i <(tar )` pattern here if we don't want a temporary file at all.